src/api/fa_hb_customer/enrollmentUsingPUT_policy.xml.tpl

```diff
@@ -14,13 +14,15 @@
     <inbound>
         <base />
         <set-variable name="v_fiscalcode" value="@((string)context.Request.Body.As<JObject>(preserveContent: true)["id"])" />
-        <set-variable name="v_vatNumbers" value="@(context.Request.Body.As<JObject>(preserveContent: true)["vatNumbers"])" />
+        <!--set-variable name="v_vatNumbers" value="@(context.Request.Body.As<JObject>(preserveContent: true)["vatNumbers"])" /-->
```

Comment:
Could you please remove commented out code?

Reply:
done